Backend/Remora.Discord.API/API/Gateway/Events/Users/UserUpdate.cs

```diff
@@ -35,7 +35,8 @@ public record UserUpdate
     Snowflake ID,
     string Username,
     ushort Discriminator,
-    string? GlobalName,
+    // GlobalName is not present in the OpenAPI spec, only in documentation and is not returned by all endpoints
```

Comment:
No need to comment the individual constructor parameters.

Reply:
removed these too